### ISSUE TITLE
Does this client support NATS streaming protocol?

### ISSUE THREAD

**Issue by pradeeproark** [joshglendenning/websocket-nats#12]
as defined in
https://nats.io/documentation/streaming/nats-streaming-protocol/

In particular I am looking for a way to specify the request the position in the stream to replay from?
https://nats.io/documentation/streaming/nats-streaming-intro/
Historical message replay by subject

**Comment by Yugloocamai** [comment]
bump, this would be nice

**Comment by xianghu1314** [comment]
yes,i very need it!  i dont know what the client's nats-streaming-protocol meaning
